packages/themes/src/themes/shadcn.css

```
@@ -0,0 +1 @@
+@source "./dist/themes/shadcn.js";
```

I think this needs to be:

```suggestion
@source "./shadcn.js";

```

Otherwise, the relative path doesn't resolve to anything